docs/content/integrations/snowflake.mdx

```diff
@@ -1,230 +1,33 @@
 ---
-title: "Using Dagster with Snowflake | Dagster Docs"
+title: "Snowflake + Dagster"
 description: Store your Dagster assets in Snowflake
 ---
 
-# Using Dagster with Snowflake
+# Snowflake + Dagster
 
-This tutorial focuses on how to store and load Dagster's [software-defined assets (SDAs)](/concepts/assets/software-defined-assets) in Snowflake.
+Using Dagster's [software-defined assets](/concepts/assets/software-defined-assets) and Snowflake I/O manager, you can easily interact with Snowflake tables alongside other Dagster assets.
 
-By the end of the tutorial, you will:
-
-- Configure a Snowflake I/O manager
-- Create a table in Snowflake using a Dagster asset
-- Make a Snowflake table available in Dagster
-- Load Snowflake tables in downstream assets
+Managing your Snowflake tables with Dagster enables you to:
```

Comment:
having a bit of writers block with this list/the intro in general. planning to circle back to it in a half hour or so with fresh eyes, but if you have any topics you think would be good to mention i'm happy to take suggestions!